indexer/src/utils/filter.ts

```diff
@@ -0,0 +1,62 @@
+// Starknet
+import { Transaction, Event } from "../deps.ts";
+
+const KAKAROT_ADDRESS = Deno.env.get("KAKAROT_ADDRESS");
+if (KAKAROT_ADDRESS === undefined) {
+  throw new Error("ENV: KAKAROT_ADDRESS is not set");
+}
+
+export const isKakarotTransaction = (transaction: Transaction) => {
```

Comment:
I see there are unit tests written for the other `ethValidationFailed` function, maybe as a follow up we could have a good first issue with unit tests also for this function, wdyt?

Reply:
https://github.com/kkrt-labs/kakarot-rpc/issues/1117